### PR TITLE
Prepare RC branch for release

### PR DESCRIPTION
**Pull request by GlennBullingham**

### CODE REVIEW

**Comment by GlennBullingham** [comment]
bors try

**Comment by tiagolobocastro** [comment]
bors try

File: deploy/core-agents-deployment.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ spec:
             requests:
               cpu: 500m
               memory: 16Mi
-          image: mayadata/mcp-core:release-1.0.2
+          image: mayadata/mcp-core:v1.0.2
```
Comment (tiagolobocastro) [inline deploy/core-agents-deployment.yaml:46]: You don't need to modify these yourself, change the release tag on "./scripts/check-deploy-yamls.sh" and then the build should pass.

**Comment by GlennBullingham** [comment]
bors try

**Review by blaisedias — APPROVED** [review]

**Comment by GlennBullingham** [comment]
bors merge

**Pull request merged**